oracle-plugin/src/e2e-test/java/io.cdap.plugin/oracle/stepsdesign/Oracle.java

```diff
@@ -37,24 +37,29 @@ public void clickOnPreviewDataForOracleSink() {
     openSinkPluginPreviewData("Oracle");
   }
 
-  @Then("Get count of no of records transferred to target Oracle Table")
-  public void getCountOfNoOfRecordsTransferredToTargetOracleTable() throws SQLException, ClassNotFoundException {
+  @Then("Validate records transferred to target table is equal to number of records from source table")
```

Comment:
This is not required to be performed explicitly instead this can be done internally while matching the column values.

Reply:
Accommodated this internally in the function to match column values.